features/netsocket/WiFiSoftAPInterface.h

```diff
@@ -0,0 +1,110 @@
+
+/* WiFiSoftAPInterface
```

Comment:
See my previous comments - I don't think we're ready to standardise/document/test this as mbed OS API at this point, which is what this header file being here would suggest.

But you're free to provide the API locally.